src/vs/workbench/contrib/debug/browser/debugCommands.ts

```diff
@@ -711,7 +711,7 @@ CommandsRegistry.registerCommand({
 
 CommandsRegistry.registerCommand({
 	id: SELECT_AND_START_ID,
-	handler: async (accessor: ServicesAccessor, debugType: string | unknown) => {
+	handler: async (accessor: ServicesAccessor, debugType: string | unknown, noDebug?: boolean) => {
```

Comment:
Please put this in an options object like `{ noDebug?: boolean }` so that it can be extended, thanks